common/views/components/VenueHours/VenueHours.js

```diff
@@ -171,47 +167,60 @@ const VenueHours = ({ venue, weight }: Props) => {
           ))}
         </ul>
       </VenueHoursTimes>
-      {upcomingExceptionalPeriod && upcomingExceptionalPeriod.length > 0 && (
-        <JauntyBox
-          topLeft={randomPx()}
-          topRight={randomPx()}
-          bottomRight={randomPx()}
-          bottomLeft={randomPx()}
-        >
-          <h3
-            className={classNames({
-              [font({ s: 'HNM4' })]: true,
-            })}
-          >
-            <div
-              className={classNames({
-                'flex flex--v-center': true,
-              })}
+      {upcomingExceptionalPeriods.map(upcomingExceptionalPeriod => {
+        const firstOverride = upcomingExceptionalPeriod.find(
+          date => date.overrideType
+        );
+        const overrideType =
+          firstOverride && firstOverride.overrideType === 'other'
+            ? 'Unusual'
```

Comment:
Did you check that ‘unusual’ is the right word with the content team? Looks strange to me (although can’t think what would be better).